core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -128,7 +140,7 @@ protected String name() {
         }
       }
     } catch (IOException ioe) {
-      throw new RuntimeException("Failed to list tables under " + namespace, ioe);
+      throw new RuntimeIOException(ioe, "Failed to list tables under: %s ", namespace);
```

Comment:
Please remove the trailing space in this error message.

Reply:
OK